src/containers/ExpandedPostModal.js

```diff
@@ -10,19 +10,28 @@ import Post from '../components/Post'
 export default class ExpandedPostModal extends React.Component {
   static propTypes = {
     dispatch: func,
+    socket: object,
     commentId: string,
     post: object
   }
 
   componentDidMount () {
-    const { commentId, post: { id }, dispatch } = this.props
+    const { commentId, post: { id, numComments }, dispatch, socket } = this.props
     dispatch(fetchComments(id, {offset: 3}))
     .then(() => {
       if (commentId) {
         const wrapper = document.getElementById(modalWrapperCSSId)
         scrollToComment(commentId, wrapper)
       }
     })
+    socket.on(`p/${id}/comment_added`, function (){
+      dispatch(fetchComments(id, {offset: numComments, refresh: true}))
```

Comment:
If only subscribers to the specific comment thread get updates, we might as well send the whole comment down with the socket event, to save a round trip. If we have to broadcast to everyone, then maybe not.

Which brings up a question about security: we should make sure that only users who pass the checkAndSetPost policy can subscribe to updates for a given post. Hopefully Sails lets you use the same policies for sockets?